about.html

```diff
@@ -0,0 +1,114 @@
+        <ul class="flex-list">
+          <li><a href=""></a><i class="fa-brands fa-facebook-f"></i></li>
+          <li><a href=""></a><i class="fa-brands fa-twitter"></i></li>
+          <li><a href=""></a>English</li>
+          <li><a href=""></a>My Page</li>
+          <li><a href=""></a>Logout</li>
+        </ul>
+      </nav>
+      <nav class="desktop-nav">
+        <a href="index.html" class="nav-logo" ><img src="./images/logo-small.png" alt=""></a>
+        <ul class="flex-list close-menu">
+          <li><a href="about.html">About</a></li>
+          <li><a href="index.html">Register</a></li>
+          <li><a href="#about-school">Mission</a></li>
+          <li><a href="#testimonials">Facts</a></li>
+          <li class="last-child bubbly-button"><a href="#"></a>Killy Int'l</li>
+        </ul>
+      </nav>
+    </header>
+    <!-- Headline Section -->
+    <section id="headline-section">
+      <h1 class="intro-heading">LEARN THE PRACTICAL WAY</h1>
+
+      <div id="intro-text">
+        <p>
+          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Tempore,
+          assumenda aperiam corporis nemo rem esse sed repellat expedita est
+          accusantium sit dignissimos, libero iste non quis dolorem sapiente
+          saepe autem.
+        </p>
+      </div>
+      <!-- <img src="./images/img2.png " alt="" /> -->
+      <div id="message-email">
+        <p>Please contact us via email for any questions or enquiries</p>
+        <h5 id="email">killynelly@gmail.com</h5>
+      </div>
+    </section>
+    <!-- About School Section -->
+    <section id="about-school">
+      <h2 id="about-heading">About School</h2>
+      <div class="about-intro">
+        <p>
+          Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aliquam iste
+          ducimus, officiis dicta eligendi ad beatae non itaque corrupti alias
+          natus quo saepe sit fugiat at repellendus consequatur. Corrupti, ex.
+        </p>
+      </div>
+      <div class="logo">
+        <img src="./images/school-logo.png" alt="" />
+      </div>
+    </section>
+    <!--Standout Section  -->
+    <section id="standout-section">
+      <h2 class="section-intro">Let Your Kid Standout</h2>
+      <hr class="orange-hr" />
+    </section>
+    <!-- Testimonials -->
+    <section id="testimonials">
+      <h2 class="testimonials-heading">Important Facts</h2>
+      <hr class="orange-hr" />
+      <p class="about-intro">
+        Take a look at some of the important events and facts about our school
+      </p>
+      <div id="facts-container">
+        <!-- Fact Card One -->
+        <div class="fact-card" id="fact-one">
+          <h2>1999</h2>
+          <p>Our Scool was established</p>
+        </div>
+        <!-- Fact Card Two -->
+        <div class="fact-card" id="fact-two">
+          <h2>2005</h2>
+          <p>Best child education center</p>
+        </div>
+      </div>
+    </section>
+    <!--Footer -->
+    <footer id="footer">
```

Comment:
- [ Optional ]  The footer in the desktop version about page has the logo on the left column and the description text on the right in one row.
#### Design
![image](https://user-images.githubusercontent.com/44057863/169669235-c1d0c802-2622-4fb0-8845-6e3441b6b6f6.png)
#### Impl
![image](https://user-images.githubusercontent.com/44057863/169669226-90967c7a-0e9c-4cbf-bde7-63eaa2fa78a0.png)

- [ Optional ] In mobile version the layout remains the same as the desktop, but the background color become `white`. And the partner's sectoin becomes visible as well.
#### Footer About page - Mobile
![image](https://user-images.githubusercontent.com/44057863/169669314-417460a3-befd-4551-9851-7bc64632ad76.png)